include/reference_configs/Homo_sapiens.yaml

```diff
@@ -91,6 +91,7 @@ references:
         indexes:
           - 'salmon'
           - 'kallisto'
+      blacklist: 'https://github.com/Boyle-Lab/Blacklist/blob/master/lists/hg19-blacklist.v2.bed.gz'
```

Comment:
Can you add some documentation about this (what it does, how to get it for species of interest)?

Reply:
Thank you for pointing it out. I'm deleting it. My current update doesn't have blacklist filtering.